lib/ansible/plugins/connection/winrm.py

```diff
@@ -249,8 +249,8 @@ def _reset(self):  # used by win_reboot (and any other action that might need to
 
     def exec_command(self, cmd, in_data=None, sudoable=True):
         super(Connection, self).exec_command(cmd, in_data=in_data, sudoable=sudoable)
-        cmd_parts = shlex.split(to_bytes(cmd), posix=False)
-        cmd_parts = map(to_text, cmd_parts)
+        cmd_parts = shlex.split(to_text(cmd), posix=False)
+        cmd_parts = list(map(to_text, cmd_parts))
```

Comment:
Hmm, AFAIK py2.6 isn't supported by ansible anymore?  I tested the patch both on 2.7 and 3.5 and it works well.

+1 for a list comprehension.  I'm going to rewrite it so.